spec/21-http_header_format_rationale.md

```diff
@@ -98,17 +90,103 @@ configuration - certain header mutations may still be required. An example may b
 smart sampling mechanisms that rely on additional data propagation in
 `tracestate`.
 
-### Size limits
+## Size limits of `tracestate`
+
+### Total size limit
+
+The field `tracestate` opens up rich interoperability and extensibility
+scenarios for vendors. There are defined use cases for the `tracestate`
+like positioning request in multiple distributed tracing graphs and
+providing backward compatibility with older protocols. But `tracestate`
+is not limited to this use and will drive innovations and new scenarios
+going forward.
+
+Opaque nature of `tracestate` value raises many questions and biggest
```

Comment:
Replace with:

The opaque nature of `tracestate` introduces some tension between guarantees to vendors around propagating vendor-specific data with traces vs. storage and propagation constraints